src/components/Notifications/NotificationGroupCard/NotificationGroupCard.tsx

```diff
@@ -0,0 +1,115 @@
+import { useCallback, useEffect, useState } from 'react'
+import { NotificationType } from '@dcl/schemas'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { objectToSnake, toCamel } from 'ts-case-convert'
+import { Alert, Skeleton, Snackbar, Switch } from 'decentraland-ui2'
+import NotificationIcon from '../../NotificationIcon'
+import {
+  AccordionDescriptionStyled,
+  AccordionDetailsStyled,
+  AccordionStyled,
+  AccordionSummaryContainerStyled,
+  AccordionSummaryStyled,
+  AccordionTitleStyled,
+  NotificationItemContainerStyled,
+  NotificationItemTextIconContainerStyled,
+  NotificationItemTextStyled
+} from './NotificationGroupCard.styled'
+import { Props } from './NotificationGroupCard.types'
+
+export const NOTIFICATION_CARD_LOADING_TEST_ID = 'notification-card-loading-test-id'
+export const NOTIFICATION_CARD_TITLE_TEST_ID = 'notification-card-title-test-id'
+export const NOTIFICATION_CARD_DESCRIPTION_TEST_ID = 'notification-card-description-test-id'
+export const NOTIFICATION_CARD_SWITCH_TEST_ID = 'notification-card-switch-test-id'
+
+function NotificationGroupCard(props: Props) {
+  const {
+    isLoading,
+    isSavingSubscription,
+    subscriptionGroupKeys,
+    notificationTypesInGroup,
+    defaultExpanded,
+    disabled,
+    subscriptionDetails,
+    error,
+    onChangeNotificationSetting,
+    onClearChangeNotificationSettingError
+  } = props
+
+  const [showError, setShowError] = useState(false)
+
+  useEffect(() => {
+    if (error) {
+      setShowError(true)
+    }
+  }, [error])
+
+  const handleOnChangeNotificationSetting = useCallback(
+    (checked: boolean, type: NotificationType) => {
+      if (!isLoading) {
+        const subscriptionDetailsChanged = {
+          ...subscriptionDetails,
+          messageType: { ...subscriptionDetails.messageType, [toCamel(type)]: { inApp: true, email: checked } }
+        }
+        onChangeNotificationSetting(objectToSnake(subscriptionDetailsChanged))
+      }
+    },
+    [subscriptionDetails, onChangeNotificationSetting, isLoading]
+  )
+
+  const handleClose = useCallback(() => {
+    setShowError(false)
```

Comment:
There's no need to have a `showError` state, the `onClearChangeNotificationSettingError` function should clear the error in the state and the toast won't be shown anymore.
You can remove the useEffect on line 42 as well.